recipes/install-redis-server.rb

```diff
@@ -0,0 +1,6 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: install-redis-server
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+install_package('redis-server')
```

Comment:
you might want to confirm that redis is storing keys to disk periodically. I don't recall if that's the default or not.